modules/Bio/EnsEMBL/Analysis/Hive/Config/StarScallopRnaseq.pm

```diff
@@ -0,0 +1,827 @@
+    'protein_blast_index' => '' || catdir( $self->o('base_blast_db_path'), 'uniprot', $self->o('uniprot_version'), 'PE12_vertebrata_index' ),    # Indicate Index for the blast database.
+    'protein_entry_loc' => catfile( $self->o('base_blast_db_path'), 'uniprot', $self->o('uniprot_version'), 'entry_loc' ),                       # Used by genscan blasts and optimise daf/paf. Don't change unless you know what you're doing
+
+########################
+# Pipe and ref db info
+########################
+
+    'pipe_db_name' => $self->o('dbowner') . '_' . $self->o('production_name') . '_pipe_' . $self->o('release_number'),
+    'dna_db_name'  => $self->o('dbowner') . '_' . $self->o('production_name') . '_core_' . $self->o('release_number'),
+
+    'reference_db_name'   => $self->o('dna_db_name'),
+    'reference_db_server' => $self->o('dna_db_server'),
+    'reference_db_port'   => $self->o('dna_db_port'),
+
+    'star_rnaseq_for_layer_db_server' => $self->o('databases_server'),
+    'star_rnaseq_for_layer_db_port'   => $self->o('databases_port'),
+
+    'scallop_initial_db_server' => $self->o('databases_server'),
+    'scallop_initial_db_port'   => $self->o('databases_port'),
+
+    'scallop_blast_db_server' => $self->o('databases_server'),
+    'scallop_blast_db_port'   => $self->o('databases_port'),
+
+    # This is used for the ensembl_production and the ncbi_taxonomy databases
+    'ensembl_release' => $ENV{ENSEMBL_RELEASE},    # this is the current release version on staging to be able to get the correct database
+
+    databases_to_delete => ['reference_db'],
+
+########################
+# BLAST db paths
+########################
+    'base_blast_db_path' => $ENV{BLASTDB_DIR},
+
+######################################################
+#
+# Mostly constant settings
+#
+######################################################
+
+    genome_dumps => catdir( $self->o('output_path'), 'genome_dumps' ),
+    # This one is used in replacement of the dna table in the core db, so where analyses override slice->seq. Has simple headers with just the seq_region name. Also used by bwa in the RNA-seq analyses. Not masked
+    faidx_genome_file => catfile( $self->o('genome_dumps'), $self->o('species_name') . '_toplevel.fa' ),
+
+    'min_toplevel_slice_length' => 250,
+
+########################
+# Extra db settings
+########################
+
+    'num_tokens' => 10,
+
+########################
+# Executable paths
+########################
+    'star_path'    => '/homes/fergal/bin/STAR',
+    'scallop_path' => '/homes/fergal/bin/scallop',
+
+    'blast_type'             => 'ncbi',                                                                         # It can be 'ncbi', 'wu', or 'legacy_ncbi'
+    'uniprot_blast_exe_path' => catfile( $self->o('binary_base'), 'blastp' ),
+    'bwa_path'               => catfile( $self->o('software_base_path'), 'opt', 'bwa-051mt', 'bin', 'bwa' ),    #You may need to specify the full path to the bwa binary
```

Comment:
This can be removed, bwa is not needed here

Reply:
I think it's used for creating the index in "index_rnaseq_genome_file".

Reply:
it shouldn't otherwise it would create an index for bwa which star could not use

Reply:
Ok you're right it's better if it stays. The indexing is not really needed because it would be done in the load assembly pipeline. But by having it we make sure that the indexing checked so I'm happy with it staying